cluster/gce/gci/configure-helper.sh

```diff
@@ -558,6 +586,42 @@ function create-master-pki {
     PROXY_CLIENT_CERT_PATH="${pki_dir}/proxy_client.crt"
     write-pki-data "${PROXY_CLIENT_CERT}" "${PROXY_CLIENT_CERT_PATH}"
   fi
+
+  if [[ ! -z "${PROXY_SERVER_CA_CERT:-}" ]]; then
+    mkdir -p "${pki_dir}"/proxy-server
+    PROXY_SERVER_CA_KEY_PATH="${pki_dir}/proxy-server/ca.key"
+    write-pki-data "${PROXY_SERVER_CA_KEY}" "${PROXY_SERVER_CA_KEY_PATH}"
```

Comment:
Does PROXY_SERVER_CA_KEY need to be stored on the master in gke? That's probably not something we want to do.

Reply:
Agreed. Fixed.